failed to compile on macOS

```
   Compiling frust v0.1.0 (https://github.com/tbehner/frust.git#cbcc8f5f)
error[E0432]: unresolved import `std::os::linux::fs::MetadataExt`
  --> /Users/messense/.cargo/git/checkouts/frust-86b34f6086ce95f2/cbcc8f5/src/formatter.rs:13:5
   |
13 | use std::os::linux::fs::MetadataExt;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Could not find `linux` in `os`

error: aborting due to previous error(s)

error: failed to compile `frust v0.1.0 (https://github.com/tbehner/frust.git#cbcc8f5f)`, intermediate artifacts can be found at `/var/folders/x5/5tc76rdj7yq_7cbc4g1mpn9c0000gn/T/cargo-install.xfa1MqkCNjz7`

Caused by:
  Could not compile `frust`.

To learn more, run the command again with --verbose.
```

--- a/src/formatter.rs
+++ b/src/formatter.rs
@@ -9,8 +9,7 @@ use std::time;
 use chrono::{Local, TimeZone};
 use ColorConfig;
 use std::ffi::OsStr;
-use std::os::unix::fs::FileTypeExt;
-use std::os::linux::fs::MetadataExt;
+use std::os::unix::fs::{FileTypeExt, MetadataExt};
 
 struct RgbColor {
     red: u8,

